.github/workflows-source/ci-main.yml

```diff
@@ -113,19 +113,22 @@ jobs:
         uses: actions/upload-artifact@v4
         with:
           name: bazel-targets
-          retention-days: 1
+          retention-days: 30
```

Comment:
Hey I think we'll need to be a little careful here, as we don't have unlimited storage. Separately, isn't the build profile based on data in ceph so we'd want to actually increase the retention days there?

Reply:
I'll change it to 14 days and we can see latter on how much roughly that contributes. Leaving retention to 1 day in CEPH is OK - if we see non-deterministic build, we can pull artifacts from CEPH immediately and investigate. I've aligned it here to be the same as for `bazel-bep.pb` and `profile.json` as the idea is to also save the targets that were modified in the PR along with all the build stats.